runtime/assembly/storage.ts

```diff
@@ -0,0 +1,34 @@
+import {Utils} from './utils';
```

Comment:
should we create a new folder for storage? or it could just stay in the main folder?

Reply:
I thought the same thing.
My first suggestion was to create `modules` folder and place the `Storage` there, but I wasn't sure. I decided to keep the storage file in the main folder for now until we add a couple more files, then we might be able to better determine their location, but if you like my `modules` suggestion, we can do it right away.
What do you think?

Reply:
yeah, `modules` sounds good